src/projectscene/view/trackspanel/trackslistmodel.cpp

```diff
@@ -32,8 +32,16 @@ TracksListModel::TracksListModel(QObject* parent)
         onSelectedTracks(tracksIds);
     });
 
-    selectionController()->dataSelectedOnTracksSelected().onReceive(this, [this](trackedit::TrackIdList tracksIds) {
-        if (!tracksIds.empty()) {
+    selectionController()->dataSelectedStartTimeChanged().onReceive(this, [this](trackedit::secs_t begin) {
+        Q_UNUSED(begin);
+        if (selectionController()->timeSelectionIsNotEmpty()) {
+            m_audioDataSelected = true;
```

Comment:
I looked and it seems to me that this `m_audioDataSelected` variable could disappear and query `selectionController()->timeSelectionIsNotEmpty()` where its value is read.

Reply:
I'd like to get rid of it too but for now it's "not possible", meaning: it will break sync of track headers and tracks in the clips view. I will address it anyway as I believe Sergey will find out what's wrong with it but I need to extend muse framework first. As this may take some additional time until this thing gets merged into musescore I suggest we merge it for now as is. Added appropriate comment in the code though